ros2topic/ros2topic/verb/pub.py

```diff
@@ -98,6 +98,10 @@ def add_arguments(self, parser, cli_name):
             '--keep-alive', metavar='N', type=positive_float, default=0.1,
             help='Keep publishing node alive for N seconds after the last msg '
                  '(default: 0.1)')
+        parser.add_argument(
+            '--yaml-file', type=str, default=None,
+            help='YAML file that has message contents, prevails <values>. '
```

Comment:
```suggestion
            help='YAML file that has message contents. '
```

Can we put `--yaml-file` and `values` into a mutually exclusive parser group?  That way it will be enforce by argparse.

Reply:
fixed in https://github.com/ros2/ros2cli/pull/925/commits/853e7f2720510c1e0279a1754d4fe0fd61837971, this is much better. thanks!